modules/Makefile

```diff
@@ -78,6 +78,7 @@ MODULES_TO_DOCUMENT = \
 	standard/IO.chpl \
 	standard/LinkedLists.chpl \
 	standard/List.chpl \
+  standard/Lists.chpl \
```

Comment:
lol, indentation XD

Reply:
Noooooooooo!

Reply:
I seriously need to force my vim to use true tabs when working on Makefiles...